bin/GameIndex.dbf

```diff
@@ -4617,14 +4618,8 @@ Region = NTSC-U
 Serial = SCUS-97150
 Name   = Formula One 2001
 Region = NTSC-U
-Compat = 3
+Compat = 5
 XgKickHack = 1
-[patches = 272F1C14]
-	comment=patches by Plot and Ref Skip videos
-	patch=0,EE,0010aafc,word,00000000
-	patch=0,EE,0010ae2c,word,00000000
-	patch=0,EE,00230010,word,00000000
-[/patches]
```

Comment:
skip video patches aren't needed anymore and game was fixed in https://github.com/PCSX2/pcsx2/commit/ad61503d8311f113ab1fc07d3955f9f576c4f11e